src/status_ctrl.js

```diff
@@ -306,7 +306,8 @@ export class StatusPluginCtrl extends MetricsPanelCtrl {
 		} else if((this.series == undefined || this.series.length == 0) && this.panel.isGrayOnNoData) {
 			this.$panelContainer.addClass('no-data-state');
 		} else {
-			this.$panelContainer.addClass('ok-state');
+		    if (!this.panel.useDefaultBackground)
+		        this.$panelContainer.addClass('ok-state');
```

Comment:
This causes a bug with the Flip feature.
if the flip feature is on and your feature is on, the panel appears empty.
Please fix it. 😄